cluster-autoscaler/hack/list-owners.py

```diff
@@ -0,0 +1,24 @@
+""" Python script to list all OWNERS of various parts of Cluster Autoscaler.
```

Comment:
Imo, before this, you have to write the following thing:
```
#!/usr/bin/env python3

# Copyright 2017 The Kubernetes Authors.
#
# Licensed under the Apache License, Version 2.0 (the "License");
# you may not use this file except in compliance with the License.
# You may obtain a copy of the License at
#
#     http://www.apache.org/licenses/LICENSE-2.0
#
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS,
# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
# See the License for the specific language governing permissions and
# limitations under the License.
```
I updated the file with the above lines and ran the script (verify-boilerplate.sh which failed) and it's working fine for me.

Reply:
Yup, thanks, updated.